src/Microsoft.AspNetCore.SignalR.Redis/RedisHubLifetimeManager.cs

```diff
@@ -551,7 +551,7 @@ public override void WriteLine(string value)
             }
         }
 
-        public class RedisExcludeClientsMessage : InvocationMessage
+        public class RedisExcludeClientsMessage : NonStreamInvocationMessage
```

Comment:
Unrelated to your change, but I just noticed this, could you change this class to private